docs/ABI/Mangling.rst

```diff
@@ -1041,3 +1041,19 @@ Some kinds need arguments, which precede ``Tf``.
 If the first character of the string literal is a digit ``[0-9]`` or an
 underscore ``_``, the identifier for the string literal is prefixed with an
 additional underscore ``_``.
+
+Conventions for constructing names
+----------------------------------
+
+Swift is a language that integrates with multiple languages (C, C++, ObjC,
+ObjC++). To achieve interoperability symbols from other languages have to be
+adapted in order to be integrated into Swift.
+
+Importing C and C++ structs
+~~~~~~~~~~~~~~~~~~~
+
+C and C++ structs use their original names and their owning modules. Since Swift
```

Comment:
> their original names and their owning modules

The two parts of this conjunct seem disconnected from one another. Perhaps we're missing an example, or perhaps it's just enough to leave it at "we don't touch the mangling of the original symbol".

Reply:
Went with "we don't touch the mangling of the original symbol". Thanks!